src/components/layer-controller/LayerOptions.js

```diff
@@ -7,6 +7,14 @@ import Select from '@material-ui/core/Select';
 
 import { COLORMAP_OPTIONS } from '../utils';
 
+const DOMAIN_OPTIONS = ['Full', 'Min/Max'];
```

Comment:
Better names? I'm guessing `Full` is `[0, dtype.max]` and `Min/Max` are data-specific but that's not totally clear.

Reply:
This is what OMERO calls it.  Do you have any ideas?  Perhaps showing the datatype, like `uint16`?